README.md

```diff
@@ -14,30 +14,33 @@ in a Python package.
 
 ## Installation
 
+### Notebook
+
 To install this extension into JupyterLab, do the following:
 
 ```bash
 jupyter labextension install @jupyterlab/hub-extension
 ```
 
+### JupyterHub
+
 In `jupyterhub_config.py` configure the Spawner to tell the single-user notebook servers to default to Jupyter-Lab:
 
 ```
-c.Spawner.default_url = '/lab'
+c.Spawner.environment = { 'JUPYTER_ENABLE_LAB': 'yes' }
```

Comment:
I'm wary of this change.  It is specific to the docker-stacks setup.  Could we perhaps have this as an either-or with a link back to docker stacks for the environment variable usage?

Reply:
sure, now the docker-stack change landed, I can link to the usage.

would it work to list the `default_url` appraoch alongside running jupyter-labhub?  But that along wouldn't include the hub menu item of this extension, right?

Reply:
The administrator would need to install this explicitly and use the `default_url` approach if not using the docker stack image.